pkgs/top-level/all-packages.nix

```diff
@@ -19578,6 +19578,8 @@ with pkgs;
 
   EmptyEpsilon = callPackage ../games/empty-epsilon { };
 
+  endgame-singularity = callPackage ../games/endgame-singularity { };
+
```

Comment:
Same here: @domenkozar should this be in `python-packages`?

Reply:
Same here too :)

Reply:
Actually I thought the same applied for the `python2Packages`←→`python.pkgs`+`disabled=!isPy3` thing, but apparently I was wrong?

Reply:
Very possible I'm the one who is wrong, @domenkozar should know better.

Reply:
Same here, I actually meant to ping you @dotlambda. Somehow the matching first two letters got me confused :grin:

Reply:
@fgaz is right.

Reply:
Singularity is taken:
```
Attribute name: nixpkgs.singularity
Package name: singularity
Version: 2.5.1
Description: Designed around the notion of extreme mobility of compute and reproducible science, Singularity enables users to have full control of their operating system environment
```

For the same reason I also changed the exe name (not that bad since an exe name equal to the pkg name is not surprising). Should I revert it to `singularity`?

Reply:
No, seems reasonable.